mobile/src/home-view.js

```diff
@@ -217,18 +218,21 @@ class HomeView extends PureComponent {
               </View>
               {this.renderSearch()}
               {leads.length === 0 && <Text style={s.noConnections}>{t('no_connections')}</Text>}
-              {leads.map((card, index) => (
-                <CardListItem
-                  key={index}
-                  showExpanded={index === selectedCard}
-                  showCard={() => this.showCard(index)}
-                  showAlert={() => this.showAlert()}
-                  onUpdateNotes={notes => this.updateScannedCard(index, { ...card, notes })}
-                  user={card}
-                  primaryColor={primaryColor}
-                  {...card}
-                />
-              ))}
+              <FlatList
+                data={leads}
+                renderItem={({ item, index }) => (
+                  <CardListItem
+                    key={index}
```

Comment:
instead of specifying the `key` on the rendered item, it needs to come from the `keyExtractor` passed to the FlatList